package.json

```diff
@@ -45,19 +44,18 @@
     "cross-env": "^1.0.8",
     "css-loader": "^0.23.1",
     "eslint": "^2.5.3",
+    "file-loader": "0.10.1",
     "html-webpack-plugin": "^2.14.0",
     "source-map-loader": "^0.1.5",
-    "superstatic": "^4.0.2",
     "url-loader": "0.5.8",
-    "file-loader": "0.10.1",
-    "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
+    "webpack": "^1.12.14"
   },
   "dependencies": {
     "css-loader": "^0.23.1",
     "express": "^4.14.0",
+    "onfido-sdk-ui": "^1.1.0",
```

Comment:
Make the version fixed so that we don't break sample app with every onfido-sdk-ui release update